GetStream/stream-chat-swift
# Add mute expiration support when muting a channel

### 🔗 Issue Links

Resolves https://github.com/GetStream/stream-chat-swift/issues/2807
Relates https://github.com/GetStream/ios-issues-tracking/issues/762

### 🎯 Goal

Add mute expiration support when muting a channel

### 🧪 Manual Testing Notes

- Open a ChannelList
- Swipe a channel
- Tap the Debug Button
- Tap "Mute channel with expiration" button
- Set up an expiration duration (e.g.: 10000)
- Write a message to this channel from a different account
- Unread count bubble does not appear
- Wait expiration duration
- Write one more message
- Unread count bubble should appear

### ☑️ Contributor Checklist


### 🎁 Meme

[!gif](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExa2lxcGIzMnJlMWZudzUxZmRscDRtN2x2bGo0cjhpZmFudml1cnlzMCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/swtiK9jRfE0zS/giphy.gif)
